docker-compose.build.yml

```
@@ -3,10 +3,11 @@ services:
   nginx:
     build: ./nginx
   api:
-    build: .
+    build:
+      target: api
```

```suggestion
      context: .
      target: api
```

Unable to run `docker compose` without the `context` field

---

huh, I had the issue the other way around (when context had subargs, target was needed)

---

might be that here.. it does not have subargs :)